Packs/XSOARStorage/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "XSOAR Storage",
+    "description": "XSOAR Storage provides a server wide Key/Value store.\n",
```

Comment:
We can add more details here
"XSOAR Storage provides a server-wide Key/Value store, where data can be stored or retrieved, and is separated by using namespace parameter.\n "